protected/humhub/modules/admin/views/setting/statistic.php

```diff
@@ -28,4 +30,32 @@
 
 <?php ActiveForm::end(); ?>
 
+<script>
+
+    let initCodeMirrorTimeout = setTimeout(initCodeMirror, 200);
```

Comment:
Hm, I would be very uncomfortable working with the timeouts.

Maybe this event do the trick?
https://github.com/humhub/humhub/blob/master/static/js/humhub/humhub.client.pjax.js#L44